packages/core/src/mixins/properties-component.js

```diff
@@ -13,7 +13,7 @@ import { prop, Property } from '../lib/property.js';
  * @return {*} The parsed value.
  */
 function getValue(property, attrVal) {
-    if (attrVal === '' && property.accepts(Boolean)) {
+    if ((attrVal === '' || property.name === attrVal) && property.accepts(Boolean)) {
```

Comment:
In DNA the property name could not be the same as the bound attribute, eg:
`'message': prop(String).attribute('msg')`

Reply:
We should use `(property.attrName || property.name)`